cypress/integration/specs/nav.spec.ts

```diff
@@ -0,0 +1,143 @@
+import {NavbarPage} from '../../pages/navbarPage';
+
+describe('Navbar', () => {
+  before(() => {
+    NavbarPage.visitHome();
+  });
+  describe('Navbar Cluster Items', () => {
+    it('cluster', () => {
+      NavbarPage.clickItem('#nav-cluster');
+      NavbarPage.assertUrlContains('cluster');
+    });
```

Comment:
nit: Can you separate tests with new lines?

Reply:
Made the required changes.